torchvision/models/shufflenetv2.py

```diff
@@ -162,7 +162,7 @@ def _shufflenetv2(arch: str, pretrained: bool, progress: bool, *args: Any, **kwa
     if pretrained:
         model_url = model_urls[arch]
         if model_url is None:
-            raise NotImplementedError(f"pretrained {arch} is not supported as of now")
+            raise ValueError(f"No checkpoint is available for model type {arch}")
```

Comment:
Just an unrelated correction, to align the exception type and value with the entire TorchVision so that we can capture it easier.